src/ui/__fixtures__/signTransactionFix.ts

```diff
@@ -1,22 +1,15 @@
-import { SignTransaction } from "../pages/SidePage/components/IncomingRequest/components/SignTransactionRequest.types";
+import {
+  PeerConnectSigningEvent,
+  PeerConnectSigningEventTypes,
+} from "../../core/cardano/walletConnect/peerConnection.types";
 
-const signTransactionFix: SignTransaction = {
-  action: "CAST_VOTE",
-  actionText: "Cast Vote",
-  data: {
-    id: "2658fb7d-cd12-48c3-bc95-23e73616b79f",
-    address: "stake_test1uzpq2pktpnj54e64kfgjkm8nrptdwfj7s7fvhp40e98qsusd9z7ek",
-    event: "CF_TEST_EVENT_01",
-    category: "CHANGE_SOMETHING",
-    proposal: "YES",
-    network: "PREPROD",
-    votedAt: "40262406",
-    votingPower: "10444555666",
+const signTransactionFix: PeerConnectSigningEvent = {
+  type: PeerConnectSigningEventTypes.PeerConnectSign,
+  payload: {
+    identifier: "EN5dwY0N7RKn6OcVrK7ksIniSgPcItCuBRax2JFUpuRc",
+    payload: "Hello",
```

Comment:
I think we should have 1 for text like "Hello" and another for some JSON and have a test for each